src/middleware.js

```diff
@@ -0,0 +1,19 @@
+'use strict';
+
+const db = require('./db');
+
+exports.saveContactFormData = function(req, res, next) {
+  const { name, email, subject, message } = req.body;
+
+  db.knex
+    .table('contacts')
+    .insert({ name, email, subject, message })
+    .then((result) => {
+      console.log(result);
+    next();
```

Comment:
Fix the indentation here...this `next()` call is happening inside the function that you pass to `then()`.